src/main/java/uk/gov/indexer/dao/IndexedEntriesUpdateDAO.java

```diff
@@ -16,4 +16,7 @@
 
     @SqlQuery("SELECT MAX(SERIAL_NUMBER) FROM " + INDEXED_ENTRIES_TABLE)
     int lastReadSerialNumber();
+
+    @SqlBatch("INSERT INTO " + INDEXED_ENTRIES_TABLE + "(SERIAL_NUMBER, ENTRY) VALUES(:serial_number, :content)")
+    void writeBatch(@BindBean Iterator<OrderedIndexEntry> orderedIndexEntry);
```

Comment:
Why use `Iterator` when [you can use `Iterable`](http://jdbi.org/sql_object_api_batching/) and pass a `List` straight in?